cdk/src/open-data-platform/data-plane/schema/schema.sql

```diff
@@ -34,27 +32,45 @@ CREATE TABLE IF NOT EXISTS demographics(
 
 ALTER TABLE demographics
     -- SRID 4326 maps the shape to latitude and longitude.
-    ALTER COLUMN geom TYPE GEOMETRY(Geometry, SRID_LAT_LONG);
-
+    ALTER COLUMN geom TYPE GEOMETRY(Geometry, 4326);
 
 CREATE INDEX IF NOT EXISTS geom_index
     ON demographics
     USING GIST (geom);
 
+-- Water-system-level data
+
 CREATE TABLE IF NOT EXISTS water_systems(
     pws_id varchar(255) NOT NULL,
     lead_connections_count real,
     PRIMARY KEY(pws_id)
     );
 
 ALTER TABLE water_systems
-    ADD COLUMN IF NOT EXISTS geom GEOMETRY(Geometry, SRID_LAT_LONG);
+    ADD COLUMN IF NOT EXISTS geom GEOMETRY(Geometry, 4326);
 
 CREATE INDEX IF NOT EXISTS geom_index
     ON water_systems
     USING GIST (geom);
 
--- Roles and Grants
+-- Parcel-level data
+
+CREATE TABLE IF NOT EXISTS parcels (
+    id serial PRIMARY KEY
+);
+ALTER TABLE parcels
+    -- TODO: consider standardizing addresses into a PostGIS type once we have data.
+    -- https://postgis.net/docs/manual-2.5/Address_Standardizer.html
+    ADD COLUMN IF NOT EXISTS address text,
```

Comment:
How come these statements can't be in the CREATE TABLE section?

Reply:
No reason. I've merged them.